test/hotspot/jtreg/runtime/jni/loadLibraryUnicode/LoadLibraryUnicode.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2021 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
```

Comment:
The test sources have Apache license, I thought we always used GPL for tests.

Reply:
I'm sure you are right. Corrected copyright headers in all the test files.